test/Utils_t.py

```diff
@@ -219,7 +219,7 @@ def test_gfal_copy(self):
         outfile = "{0}/{1}".format(basedir,outname)
         outfilestore = outfile.replace("/hadoop/cms", "")
```

Comment:
Does this need to be /ceph/ too?

Reply:
yeah more replacements are needed

Reply:
Gotcha, I think the main places are in the unit tests and whichever executable is used in `metis/executables/` (unless you have a custom one). Can't hurt to update `CMSSWTask` too, as it inherits from `CondorTask`



ceph...exciting :)